src/Assets/Styles/App.scss

```diff
@@ -8,6 +8,8 @@ html {
 body {
   height: 100%;
   margin-bottom: 60px;
+  background-image: url('../Images/background.jpeg') !important;
```

Comment:
don't use !important, you can make the rule more specific instead, like so:
`html body {
  background: no-repeat url('../Images/background.jpeg');
  background-size: cover;
}`